presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -75,9 +75,12 @@ public class TaskManagerConfig
     private Integer partitionedWriterCount;
     private int taskConcurrency = 16;
     private int httpResponseThreads = 100;
+    private int taskCreationTaskCountBackpressureThreshold = httpResponseThreads;
+    private DataSize taskCreationFreeMemoryLowBackpressureThreshold = new DataSize(32, Unit.MEGABYTE);
```

Comment:
I am suspicious of this value.  It's either way too large or way too small.  Overall, I wonder how feasible it is in the JVM to attempt to judge how much memory is sufficient to begin backpressure specifically for new task creation.